File: web/src/render/strikethroughRenderer.ts
import MarkdownNode from "../types/MarkdownNode";
import type Renderer from "../types/Renderer";
import type RendererState from "../types/RendererState";
import renderTag from "./renderTag";

const renderer: Renderer = {
	name: "strikethrough",
	render,
};
export default renderer;

function render(node: MarkdownNode, state: RendererState) {
	renderTag(node, state, "del");
}

File: dotnet/Allmark/Render/StrikethroughRenderer.cs
namespace Allmark.Render;

using Allmark.Types;

public static class StrikethroughRenderer
{
    public static OutputRenderer Create()
    {
        return new OutputRenderer
        {
            Name = "strikethrough",
            Render = Render,
        };
    }

    public static void Render(MarkdownNode node, RendererState state, bool? decode = true)
    {
        RenderTag.Execute(node, state, "del");
    }
}
